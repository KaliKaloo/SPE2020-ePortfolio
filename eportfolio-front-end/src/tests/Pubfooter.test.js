import React from 'react'
import { render, cleanup } from '@testing-library/react'
import '../../node_modules/bootstrap/dist/css/bootstrap.min.css';
import PubFooter from '../published/components/pubFooter';


afterEach(cleanup)

it('should take a snapshot for PubFooter UI', () => {
  const { asFragment } = render(<PubFooter />)

  expect(asFragment(<PubFooter />)).toMatchSnapshot()
})
