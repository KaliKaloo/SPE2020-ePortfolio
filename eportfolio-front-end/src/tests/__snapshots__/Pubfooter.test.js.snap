// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot for PubFooter UI 1`] = `
<DocumentFragment>
  <div
    class="footer-container"
  >
    <section
      class="footer-section"
    >
      <div
        class="footer-wrap"
      >
        <h1
          class="site-title"
        >
          ePortfolio
        </h1>
        <small
          class="website-rights"
        >
          Â©UoB
        </small>
      </div>
    </section>
  </div>
</DocumentFragment>
`;
